### DISCUSSION TITLE
`Execution Time` with bash

### DISCUSSION THREAD

**Discussion by afuschetto** [JanDeDobbeleer/oh-my-posh#3330] [Troubleshoot] [answered]
The [Execution Time](https://ohmyposh.dev/docs/segments/executiontime) segment works in a seamless manner with `zsh` but not `bash` (the execution time of the previously executed is always zero).

The documentation says that the `execution-time` argument containing the elapsed milliseconds has to be passed the oh-my-posh executable. Nevertheless this argument is not recognized by the executable. Any ideas?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@afuschetto is your bash up to date? Because it works here. Also, why are you looking at the CLI? That's all provided for you when using the shell init functionality.

![image](https://user-images.githubusercontent.com/2492783/211193022-98a6d12a-d34e-4b02-b4c3-666f5a44ef81.png)

**Reply by afuschetto** [reply]
Good point @JanDeDobbeleer, the default version of the Bash on macOs is completely outdated!
Upgrading it, the `Execution Time` segment works like a charm. Thank you!

**Reply by JanDeDobbeleer** [reply]
@afuschetto yup, I know as I've been there myself 😅
